packages/lodestar/src/network/reqresp/reqUtils.ts

```diff
@@ -29,7 +29,7 @@ export async function sendRequest<T extends ResponseBody | ResponseBody[]>(
   signal?: AbortSignal
 ): Promise<T | null> {
   if (signal?.aborted) {
-    throw new Error("aborted");
+    throw new ErrorAborted("sendRequest aborted");
```

Comment:
This would print `Aborted sendRequest aborted`. Either modify the `ErrorAborted` constructor or this message